README.md

```diff
@@ -16,22 +15,69 @@ pip install git+https://github.com/databio/bedms.git
 
 ## Usage
 
+### Standardizing based on available schemas
+
+To choose the schema you want to standardize according to, please refer to the [HuggingFace repository](https://huggingface.co/databio/attribute-standardizer-model6). Based on the schema design `.yaml` files, you can select which schema best represents your attributes. In the example below, we have chosen `encode` schema. 
+
 ```python
 from bedms import AttrStandardizer
 
-model = AttrStandardizer("ENCODE")
+model = AttrStandardizer(
+    repo_id="databio/attribute-standardizer-model6", model_name="encode"
+)
 results = model.standardize(pep="geo/gse228634:default")
 
 assert results
 ```
 
+### Training custom schemas
+Training your custom schema is very easy with `BEDMS`. You would need two things to get started:
+1. Training Sets
+2. `training_config.yaml`
+
+To instantiate `TrainStandardizer` class:
+
+```python
+from bedms.train import AttrStandardizerTrainer
+
+trainer = AttrStandardizerTrainer("training_config.yaml")
+
+```
+To load the datasets and encode them:
+
+```python
+trainer.load_encode_data()
+```
+
+To train the custom model:
+
+```python
+trainer.training()
+```
+
+To test the custom model:
 
-To see the available schemas, you can run:
+```python
+trainer.testing()
 ```
-from bedms.constants import AVAILABLE_SCHEMAS
-print(AVAILABLE_SCHEMAS)
 
-# >> ['ENCODE', 'FAIRTRACKS', 'BEDBASE'] 
+To generate visualizations such as Learning Curves, Confusion Matrices, and ROC Curve:
 
+```python
+trainer.plot_visualizations()
 ```
-AVAILABLE_SCHEMAS is a list of available schemas that you can use to standardize your metadata.
+
+### Standardizing based on custom schema
+
+For standardizing based on custom schema, your model should be on HuggingFace. The directory structure should follow the instructions mentioned on [HuggingFace](https://huggingface.co/databio/attribute-standardizer-model6). 
+
+```python
+from bedms import AttrStandardizer
+
+model = AttrStandardizer(
+    repo_id="name/of/your/hf/repo", model_name="model/name"
+)
+results = model.standardize(pep="geo/gse228634:default")
+
+assert results
```

Comment:
Instead of `assert` could we show how maybe they could iterate over results? Then maybe include comments on what the expected output of a `print` might be? Something like:
```python
for result in results:
    print(result) # {'attr': 'genome', 'score': 0.8 }
```